packages/babel-plugin-check-es2015-constants/src/index.js

```diff
@@ -44,7 +44,15 @@ export default function({ messages, types: t }) {
                 ]),
               );
             } else if (violation.parentPath.isForXStatement()) {
-              violation.parentPath.insertBefore(throwNode);
+              // Transform single statement body into BlockStatement
+              if (!violation.parentPath.get("body").isBlockStatement()) {
```

Comment:
I think you can use `violation.parentPath.ensureBlock()` (I'm not 100% sure because I've never used it, but I found it while looking at random babel files 😆 )